charts/generic-prometheus-alerts/templates/_helpers.tpl

```diff
@@ -50,3 +51,22 @@ Selector labels
 app.kubernetes.io/name: {{ include "generic-prometheus-alerts.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
+
+{{/*
+Prometheus Rule labels
+*/}}
+{{- define "generic-prometheus-alerts.ruleLabels" -}}
+{{- $targetApplication := required "A value for targetApplication must be set" .Values.targetApplication }}
+{{- $businessUnit := required "A value for businessUnit must be set" .Values.businessUnit }}
```

Comment:
think it's safe to default this to hmpps, given its in `hmpps-helm-charts`

Reply:
yes it's already defaulted, in the `values.yaml`